style.css

```diff
@@ -0,0 +1,786 @@
+  }
+
+  .ice {
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .ice img {
+    padding-left: 15px;
+    border-radius: 100%;
+    width: 80%;
+    height: 100%;
+  }
+
+  .eminem {
+    margin-top: 15px;
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .eminem img {
+    padding-left: 15px;
+    border-radius: 100%;
+    width: 80%;
+    height: 100%;
+  }
+
+  .speaker-text {
+    width: 50%;
+  }
+
+  .speaker-text h1 {
+    font-family: "LatoWebBlack", sans-serif;
+  }
+
+  .job {
+    font-family: "LatoWeb", sans-serif;
+    font-style: italic;
+    color: #ec5242;
+  }
+
+  .small-grey-line {
+    width: 40px;
+    height: 2px;
+    background: #9e9e9e63;
+  }
+
+  .description {
+    font-family: "LatoWebThin", sans-serif;
+    font-size: medium;
+    font-weight: bold;
+  }
+
+  .button-artists {
+    display: flex;
+    background-color: white;
+    margin-top: 20px;
+    width: 90%;
+    height: 50px;
+    margin-bottom: 20px;
+    border-style: none;
+    border: 2px solid #9e9e9e63;
+    font-family: "LatoWebBold", sans-serif;
+    font-size: medium;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .button-artists img {
+    width: 25px;
+    padding-bottom: 4px;
+  }
+
+  .upperpart {
+    display: none;
+  }
+
+  .header-p-desktop {
+    display: none;
+  }
+
+  .header-day-desktop {
+    display: none;
+  }
+
+  .header-place-desktop {
+    display: none;
+  }
+
+  .footer-desktop {
+    display: none;
+  }
+
+  .link-desktop {
+    display: none;
+  }
+
+  .grid-container-desktop {
+    display: none;
+  }
+
+  .rights-desktop {
+    display: none;
+  }
+
+  .logo-desktop {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  #stickers {
+    display: none;
+  }
+
+  .upperpart {
+    display: flex;
+    flex-direction: row;
+    background-color: #272a31;
+    height: min-content;
+    align-items: center;
+    place-content: flex-end;
+    padding-right: 15%;
+  }
+
+  .upperpart img {
+    width: 30px;
+    padding-right: 20px;
+  }
+
+  .upperpart h4 {
+    color: white;
+    font-family: "LatoWebThin", sans-serif;
+    font-size: small;
+    margin-right: 25px;
+  }
+
+  .artists {
+    display: block;
+  }
+
+  .artists h2 {
+    text-align: center;
+  }
+
+  .container-speakers {
+    display: flex;
+  }
+
+  .speaker {
+    width: 50vw;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .speaker-text {
+    width: 50%;
+  }
+
+  .close-hamburger {
+    display: none;
+  }
+
+  .navbar {
+    display: none;
+  }
+
+  .background-image {
+    position: relative;
+    height: 750px;
+  }
+
+  .header-h1 {
+    color: #d72311;
+    padding-left: 15%;
+    font-family: "LatoWebBlack", sans-serif;
+    font-size: xxx-large;
+  }
+
+  .header-h2 {
+    color: #d72311;
+    padding-top: 100px;
+    padding-left: 15%;
+    font-family: "LatoWebThin", sans-serif;
+    font-size: xx-large;
+    font-weight: bold;
+  }
```

Comment:
- [ ] Ouch! this is a close one, but please make sure to stick to the given color palette as I noticed a discrepancy between your design and the Behance design as shown below. This applies to all pages where there's an issue. 
- Behance design:
  ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/df0fb825-5e0f-4aca-b0ff-2327609c1bb6)
  ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/679659e2-bc61-4cca-a44f-365964eff887)

- Your design:
  ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/542702e7-bb73-4b38-857c-7f752d4c3a4f)
  ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/7e7a4261-2df3-4760-95f4-f80e13a9c826)

- Basic color swatch: `#272A31`, `#EC5242`, `#D3D3D3`